We do not deliver some python modules. They are not necessary to build the
component.

--- a/setup.py	2024-08-26 16:08:12.948997035 +0200
+++ b/setup.py	2024-08-26 16:08:30.687314878 +0200
@@ -2,20 +2,10 @@
 
 import subprocess
 
-from build_manpages.build_manpages import (  # type: ignore
-    build_manpages,
-    get_build_py_cmd,
-    get_install_cmd,
-)
 from setuptools import setup
 
 
 setup(
-    cmdclass={
-        "build_manpages": build_manpages,
-        "build_py": get_build_py_cmd(),
-        "install": get_install_cmd(),
-    }
 )
 
 # Test we have 'paper' (from libpaper) installed
--- psutils-3.3.14/pyproject.toml
+++ psutils-3.3.14/pyproject.toml
@@ -2,7 +2,6 @@
 name = "psutils"
 version = "3.3.14"
 description = "Manipulate PDF and PostScript documents"
-license = "GPL-3.0-or-later"
 authors = [
     {name = "Reuben Thomas", email = "rrt@sc3d.org"}
 ]
@@ -37,7 +36,7 @@ test = [
 
 [build-system]
 requires = [
-    "argparse-manpage[setuptools] >= 4.2",
+    #"argparse-manpage[setuptools] >= 4.2",
     "puremagic >= 1.26",
     "pypdf >= 4.3.0",
 ]
